src/parse.y

```diff
@@ -702,8 +708,6 @@ m4_argument:
 	|
 	BACKTICK lines SINGLE_QUOTE
 	|
-	BACKTICK strings SINGLE_QUOTE { free_string_list($2); }
```

Comment:
This seems concerning.  The commit message where this line was added mentions `define(`foo', `bar_t baz_t')` as something desirable to support.  My guess is that we'd now break on that?  It seems more desirable to me to support that then bare word macros.

Reply:
Or would those parse under the lines, since they'd be interpreted as bare_word macros?

Reply:
Yes, they would parse as `lines`, which is `lines line` and `line` can be a `STRING`.